Sources/Logto/Core/LogtoCore+Fetch.swift

```diff
@@ -89,16 +89,16 @@ extension LogtoCore {
         byRefreshToken refreshToken: String,
         tokenEndpoint: String,
         clientId: String,
-        resource: String? = nil,
-        scope: ValueOrArray<String>? = nil,
+        resources: String? = nil,
+        scopes: ValueOrArray<String>? = nil,
```

Comment:
I think the types of `resources` and `scopes` should always be array of string, isn't it?

Reply:
updated